src/utils/request/onUnhandledRequest.ts

```diff
@@ -172,7 +172,10 @@ Read more: https://mswjs.io/docs/getting-started/mocks\
       break
     }
 
+    case 'bypass':
+      break
+
     default:
-      return
+      throw new Error(``)
```

Comment:
I think providing an error message here can be useful for future use.

```suggestion
      throw new Error(``)
```

Reply:
Great catch. Will provide a meaningful error message when given an unknown strategy.